packages/nimble/inst/tests/test-misc.R

```diff
@@ -2,18 +2,19 @@ source(system.file(file.path('tests', 'test_utils.R'), package = 'nimble'))
 
 context("Testing of miscellaneous functionality.")
 
```

Comment:
I've wrapped this first test in a `test_that()` as cleanup (unrelated to the issue)